pkg/controller/account/account_controller.go

```diff
@@ -145,11 +148,25 @@ func (r *ReconcileAccount) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, err
 	}
 
+	// Determine if in fedramp env
+	configMap, err := controllerutils.GetOperatorConfigMap(r.Client)
+	if err != nil {
+		log.Error(err, "Failed retrieving configmap")
```

Comment:
Wouldn't we want to cancel the reconcilation here if we can't read the configmap?